blog/wasm-day-summary-blog/index.mdx

```diff
@@ -14,17 +14,17 @@ draft: false
-## **[Welcome + Opening Remarks](https://cosmonic.com/blog/industry/kubecon-eu-wasm-day-2024#welcome--opening-remarks---divya-mohan-suse) - Divya Mohan, SUSE**
+## **[Welcome + Opening Remarks](https://cosmonic.com/blog/industry/kubecon-eu-wasm-day-2024#welcome--opening-remarks--divya-mohan-suse) - Divya Mohan, SUSE**
 
 Welcome and huge thanks to the program committee: Rishit Dagli, Ashwin Kumar Uppala, Sean Isom and Wasm Day founding co-chair (and Cosmonic CEO) Liam Randall. All session recordings can be found **[here](https://youtube.com/playlist?list=PLj6h78yzYM2MQteKoXxICTWiUdZYEw6RI&feature=shared)**
 
 ## **[Components, WebAssembly's Docker's Moment](https://www.youtube.com/watch?v=wttN69wciSM&list=PLj6h78yzYM2MQteKoXxICTWiUdZYEw6RI&index=2) - Bailey Hayes, Cosmonic[](https://cosmonic.com/blog/industry/kubecon-eu-wasm-day-2024#components-webassemblys-dockers-moment---bailey-hayes-cosmonic)**
 
-In this inspirational talk, Bailey discusses the momentous changes that combined in cloud native computing---the advent of containers alongside the digitization of industry---to create Docker's seminal moment. She explains that, with the emergence of WebAssembly, a similar paradigm shift is now taking place. In Bailey's view WebAssembly Components are the catalyst for this shift.
+In this inspirational talk, Bailey discusses the momentous changes that combined in cloud native computing--the advent of containers alongside the digitization of industry--to create Docker's seminal moment. She explains that, with the emergence of WebAssembly, a similar paradigm shift is now taking place. In Bailey's view WebAssembly Components are the catalyst for this shift.
 
-Why? Bailey believes the way we build software is broken. At the heart of the discussion is the recognition that the way we build software now creates inherent barriers to success---tiny, disparate teams and a core that is filled with spikes and barriers. This is, perhaps, exemplified best in the way we build language silos. Developers coding in different languages cannot easily share libraries and code. Components change everything because they are completely language interoperable. We can take 2 components written in 2 totally different languages and have them talk to each other with strictly-defined, high-level types. We do this with the WIT IDL.
+Why? Bailey believes the way we build software is broken. At the heart of the discussion is the recognition that the way we build software now creates inherent barriers to success--tiny, disparate teams and a core that is filled with spikes and barriers. This is, perhaps, exemplified best in the way we build language silos. Developers coding in different languages cannot easily share libraries and code. Components change everything because they are completely language interoperable. We can take 2 components written in 2 totally different languages and have them talk to each other with strictly-defined, high-level types. We do this with the WIT IDL.
```

Comment:
```suggestion
Why another paradigm shift? Bailey believes the way we build software is *still* broken. At the heart of the discussion is the recognition that the way we build software now creates inherent barriers to success--tiny, disparate teams and a core that is filled with spikes and barriers. 

A common symptom of the still-broken state of  software development  is how we often build in language silos. Developers writing code in different languages cannot easily share libraries and functionality. WebAssembly Components change everything because they are completely language interoperable. We can take 2 components written in 2 totally different languages and have them talk to each other with strictly-defined, high-level types. We can even do this in a strongly typed way with the WIT IDL.
```